### DISCUSSION TITLE
Add "active electric substation building" option

### DISCUSSION THREAD

**Discussion by matkoniecz** [streetcomplete/StreetComplete#2670] [Q&A] [answered]
I noticed that many, many `building=service` that I tag with SC could be also tagged with `power=substation` ( https://wiki.openstreetmap.org/wiki/File:Service_building.png https://wiki.openstreetmap.org/wiki/Tag:power%3Dsubstation )

Maybe adding option to tag it directly in SC would be a good idea?

**Comment by mnalis** [comment]
True, most are. Another relatively common (although much less so) alternative would be `man_made=pumping_station` (if we're going to ask _"What kind of service building is this?"_ or similar). 

One could argue it would not be too spammy even in most answers would be `power=substation` due to fact that service buildings are relatively rare (compared to other buildings). 

I'm however not convinced that most people would be able to answer it correctly - just because it has "dangerous voltage" sign does not imply it is necessarily [power=substation](https://wiki.openstreetmap.org/wiki/File:Service_building.png). After all, [pumping stations](http://3.bp.blogspot.com/-moMuEcZNJDg/UAVtLck8d1I/AAAAAAAAAYc/0mMR97crCSM/s1600/TCGCC+Pump+Station+018.JPG) also likely use dangerous voltages to run pumps, so might be marked with such signs.

**Comment by westnordost** [comment, marked as answer]
TBH I'd rather keep the building-quest to only tag buildings, nothing extra.
